code/scripting/api/objs/modelinstance.cpp

```diff
@@ -256,6 +280,31 @@ ADE_FUNC(findWorldPointAndOrientation, l_SubmodelInstance, "vector, orientation"
 	return ade_set_args(L, "oo", l_Vector.Set(pnt), l_Matrix.Set(matrix_h(&orient)));
 }
 
+ADE_FUNC(findLocalPointAndOrientation, l_SubmodelInstance, "vector, orientation, [boolean world = true]", "Calculates the coordinates and orientation in the submodel's frame of reference, of a point and orientation in world coordinates [world = true] / in the object's frame of reference [world = false]", "vector, orientation", "Vector and orientation, or empty values if a handle is invalid")
+{
+	vec3d pnt;
+	matrix orient;
+
+	submodelinstance_h* smih;
+	vec3d local_pnt;
+	matrix_h* local_orient;
+	bool fromWorld = true;
+
+	if (!ade_get_args(L, "ooo|b", l_SubmodelInstance.GetPtr(&smih), l_Vector.Get(&local_pnt), l_Matrix.GetPtr(&local_orient), &fromWorld))
```

Comment:
If this function is going from the model or global frame-of-reference to local, then the args here should be named `global_`, not `local_`.  Or maybe the `local_` and non-`local_` named variables just need to be swapped.

Reply:
Fixed the naming